exporters/tests/test_datetime.py

```diff
@@ -135,3 +137,39 @@ def test_to_epoch_from_string_bad_value(timestamps):
 def test_to_epoch_from_string_bad_arg(timestamps):
     with pytest.raises(AttributeError):
         to_epoch_from_string(timestamps)
+
+
+@pytest.mark.parametrize(
+    "metric_timestamp, current_timestamp, expected",
+    [
+        (
+            "1681826843",
+            1681828643,
+            False,
+        ),  # current_timestamp - metric_timestamp = 30 min
+        (
+            "1681826903",
+            1681828643,
+            False,
+        ),  # current_timestamp - metric_timestamp = 29 min
+        (
+            "1681826783",
+            1681828643,
+            True,
+        ),  # current_timestamp - metric_timestamp = 31 min
+        (
+            "1681828643",
+            1681828643,
+            False,
+        ),  # current_timestamp - metric_timestamp = 0 min
+        ("1681828645", 1681828643, False),  # metric_timestamp in the future
+    ],
+)
+@mock.patch("pelorus.timeutil.datetime")
+def test_sample_too_old(mock_datetime, metric_timestamp, current_timestamp, expected):
```

Comment:
I would refactor here (to improve readability and avoid duplication) to something like
```python
@pytest.mark.parametrize(
    "metric_timestamp, expected",
    [
        ("1681826783", True),  # 31 min before mocked_current_datetime
        ("1681826843", False),  # 30 min before mocked_current_datetime
        ("1681826903", False),  # 29 min before mocked_current_datetime
        ("1681828643", False),  # 0 min before mocked_current_datetime
        ("1681828645", False),  # 2 sec after mocked_current_datetime
    ],
)
@mock.patch("pelorus.timeutil.datetime")
def test_sample_too_old(mock_datetime, metric_timestamp, expected):
    mocked_current_datetime = datetime.fromtimestamp(1681828643)

    mock_datetime.now.return_value = mocked_current_datetime
    mock_datetime.fromtimestamp = datetime.fromtimestamp

    assert is_out_of_date(metric_timestamp) == expected
```

Reply:
changed.